Draft of Getting Started Docs

MaggieFero opened the pull request:
Here's my first thwack at documentation for how to get started with the Vuln Cost code extension, as described in Issue #50 . I had some trouble getting it to prompt me to Authorize again, so I'm especially interested in having somebody who just got started review this PR, because I think that part might be wrong, but I remember the options being confusing at first.

I fully expect this to need revisions before merge, but I've had that issue on hold for a long time now, so I want to get at least something in here.

MaggieFero committed:
Draft of Getting Started Docs

First thwack at documentation for how to get started with the Vuln Cost code extension. I had some trouble getting it to prompt me to Authorize again, so I'm especially interested in having somebody who just got started review this PR, because I think that part might be wrong, but I remember the options being confusing at first.

bmvermeer commented on GETTING-STARTED.MD:
```diff
@@ -0,0 +1,25 @@
+VSCode will ask if you want to open the External Link. You must open the external link to continue. 
+![Open The External Website - Vuln Cost](https://user-images.githubusercontent.com/1719499/113234978-2bd32500-9257-11eb-9d21-70a02abcfa1b.png)
+4. Click Open. 
+An **Authenticate for CLI** prompt appears. This is because the Snyk CLI tool performs the underlying lookups to determine whether your packages are known vulnerable. ![Authenticate for CLI - Vuln Cost](https://user-images.githubusercontent.com/1719499/113234976-2b3a8e80-9257-11eb-8eda-ba02a7e40569.png)
```
This is not true the CLI tool does not perform the underlying lookups.
We just reused the CLI authentication flow.

MaggieFero replied:
Ahhhhhh, okay. From the end-user perspective, it was CLI auth and appears in the Snyk web UI as a CLI client, so it seemed like this was just a different UX for the CLI. Thanks for that clarification!

bmvermeer commented on GETTING-STARTED.MD:
```diff
@@ -0,0 +1,25 @@
+# Adding the Extension
+To get started with the Vuln Cost Security Scanner extension for Visual Studio Code (VSCode), you must first install the *Vuln Cost - Security Scanner* by `snyk-security.vscode-vuln-cost` from the **Extensions Marketplace**. You may need to restart VSCode to see warnings.
```
Great call out! In some cases, people restarted VSCode multiple times before it worked. (Currently we don't know why) If there are any problems make sure VS Code is fully shut down and restarted.

bmvermeer commented on GETTING-STARTED.MD:
```diff
@@ -0,0 +1,25 @@
+# Viewing Vulnerability Details
+Click within the name of a dependency that lists a number of vulns, and a blue extension icon will appear in the left margin of VSCode. When you click the extension icon, a dropdown appears with two options: **Fix Vulns** or **Learn about this package**.  
+
+If you select **Fix Vulns**,  VSCode asks if you would like to go to the external webpage. If you haven’t authorized Snyk CLI since installing the extension, you will need to [authorize your account](#authorizing-your-account) to actually see fix options. If you have already authorized Snyk CLI, the **Output** panel lists direct and indirect vulnerabilities known to affect your package version, a ...
```
authorize your existing account, or sign up for a free account if you don't have one yet.

bmvermeer reviewed (changes requested):
Great stuff, just some minor changes

MaggieFero committed:
Merge branch 'master' into master

MaggieFero committed:
Added account creation option

Mentioned that you can create a free account, as requested by PR reviewer in https://github.com/snyk/vulncost/pull/54/commits/0a4af35fd0d0b019ff2a2eb67e8c743030d06174#r615985245

MaggieFero committed:
Clarified that this isn't actually a CLI instance

Clarified that the VulnCost Code Extension apparently isn't an instance of the CLI client, but just reused the auth and appears as a CLI client in the Web UI, as suggested by PR Reviewer in https://github.com/snyk/vulncost/pull/54/commits/0a4af35fd0d0b019ff2a2eb67e8c743030d06174#r615981180

MaggieFero commented:
Thanks for the review! I've completed the requested changes, but I'm not sure how to submit the new version for review/merge. Do you already see the new version here now, or should I do something to update the files submitted for this PR (or does it require a new PR altogether? hopefully not anything that drastic)?

Ryuno-Ki commented:
> Do you already see the new version here now

We do.

bmvermeer commented:
:tada: This PR is included in version 1.6.0 :tada:

The release is available on [GitHub release](https://github.com/snyk/vulncost/releases/tag/v1.6.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
